Exam_10.js

```diff
@@ -0,0 +1,215 @@
+// Bài 1: in một bảng nhân của một sô bất kì với các số từ 1 đến 10 và hiện kết quả
+const multiplyNumber = (n) => {
+        let result
+        for (let i = 1; i < 11; i++) {
+            result = n * i
+            console.log(`${n} x ${i} = ${result}`)
+        }
+
+    }
+    // Bài 2:Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6
+const checkEvenNumber = (n) => {
+        n = Number(n)
```

Comment:
Không nên viết như thế này
Hãy tạo 1 hàm nào đó như thế này:

```javascript
const checkIsNotANumber = input => {
  return Boolean
}
```
Và sửa dòng trên thành:
```suggestion
        if (! checkIsNotANumber(n)) return;
```